tinyagentos/services/rknn_sd_server.py

```
@@ -146,13 +151,38 @@ def _ensure_pipeline_sync():
+    import gc
+    global _pipe
+    pipe = _pipe
+    _pipe = None
+    if pipe is None:
+        return
+    for method_name in ("release", "close"):
+        method = getattr(pipe, method_name, None)
+        if callable(method):
+            try:
+                method()
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)